mantle/cmd/kola/testiso.go

```diff
@@ -137,15 +136,6 @@ func newBaseQemuBuilder() *platform.QemuBuilder {
 	builder := platform.NewMetalQemuBuilderDefault()
 	builder.Firmware = kola.QEMUOptions.Firmware
 
-	// https://github.com/coreos/fedora-coreos-tracker/issues/388
-	// https://github.com/coreos/fedora-coreos-docs/pull/46
-	builder.Memory = 4096
-	if system.RpmArch() == "s390x" {
-		// After some trial and error looks like we need at least 10G on s390x
-		// Recorded an issue to investigate this: https://github.com/coreos/coreos-assembler/issues/1489
```

Comment:
Looks like we're losing a link to this issue which isn't in the copy. Let's add that comment there?

Reply:
There's also a 10G/16G discrepancy there; I kept the larger number since I believe it's what's actually in use now.
CC @Prashanth684

Reply:
10G should be fine as that's what the pipeline uses today and it works. thanks!